openwisp_controller/subnet_division/fixtures/groups.json

```diff
@@ -0,0 +1,600 @@
+[
```

Comment:
@pandafy @codesankalp why are these fixtures needed now? We should have all the permissions for both groups already created during migrations, why do we need to duplicate this all again here? This means we will have to keep updating this file in the future, which is definitely not what the right thing to do.

Reply:
In `TransactionTestCase` the Group queryset was empty. And if I use normal `TestCase` then the subnet division rule was not created, that's why @pandafy suggested to add fixtures here.

Reply:
> We should have all the permissions for both groups already created during migrations,

This is correct. All the permissions and groups are present in the database when the first test gets executed. For the subsequent test cases, `TransactionTestCase` truncates data that is present in all the tables in the cleanup. Due to this, permissions and groups data is deleted from the database.